### PR TITLE
OCS content for DevZone

### PR DESCRIPTION
**Pull request by mulbc**
Creating a WIP so that Karan can add his content.

If you have the time, please give early feedback on how we use Katacoda.
I have also included the list of pre-fetchable container images... please add them to the `openshift-42` image

You can test here:

Prod: https://www.katacoda.com/storage/courses/persistence/
Beta: https://beta.katacoda.com/storage/courses/persistence/

### CODE REVIEW

**Comment by btannous** [comment]
So far looks good, only suggestion I'd have is asking do you need this to be a top level menu item?
I think this might fit in one of the top three at https://learn.openshift.com

**Comment by mulbc** [comment]
I reviewed all the top-level items and found no item that worked for Red Hat Storage.
Additionally we envision to create more scenarios in the near future, that would probably fill up the other top-level-items quickly.

For this particular PR, we would like to keep it this way, since we already handed in our links for DevZone... if you insist on moving it after Summit, we can definitely discuss

File: homepage-pathway.json [hunk 1]
```diff
@@ -29,6 +29,10 @@
     {
       "external_link": "https://learn.openshift.com/ai-machine-learning/",
       "title": "AI and Machine Learning on OpenShift"
+    },
+    {
+      "external_link": "https://learn.openshift.com/persistence/",
+      "title": "Adding persistence to OpenShift"
```
Comment (jankleinert) [inline homepage-pathway.json:35]: For consistency with other top-level categories, please use title case here "Adding Persistence to OpenShift".

File: persistence-pathway.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "title": "OpenShift Persistence",
```
Comment (jankleinert) [inline persistence-pathway.json:2]: This title should match whatever you used as the title for the pathway on the top-level page. So please either use "OpenShift Persistence" in both places, or "Adding Persistence to OpenShift" in both places.

**Comment by jankleinert** [comment]
@mulbc regarding the placement of this section, after Summit is over, please follow up with @jdob . He's working on a re-org/restructure of the categories to be done after Summit, and he can help make sure everything is in a logical place, and if anything does end up moving, he can also ensure entries are placed in the redirects file.

Also, before this is merged, please drop in a link to your katacoda staging account. It should be the production one, for example, katacoda.com/<yourusername> rather than beta.katacoda.com/<yourusername> so we can confirm it's working against the production 4.2 image before it's merged in.

Thanks!

**Comment by mulbc** [comment]
Thanks for the review @jankleinert - I implemented the requested title changes in 6d6d01ec3156bacda4d5a6f1c324228779caedb7

My account (org) is called storage ;)
You can test here:

Prod: https://www.katacoda.com/storage/courses/persistence/
Beta: https://beta.katacoda.com/storage/courses/persistence/

**Comment by mulbc** [comment]
@jankleinert - I just checked and the production image has not been synced with the beta image yet... thus there are no local disks in the production environment and thus the OCS deployment will not work.

**Comment by mulbc** [comment]
Rebased to origin and squashed a couple of development commits to make history nicer... This is ready for merge now :)

File: environments/openshift-4-3/master/build/6_ocs-pull.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+sudo podman pull quay.io/mulbc/ocs-operator:katacoda
+sudo podman pull rook/ceph:v1.2.4
+sudo podman pull noobaa/noobaa-operator:2.0.9
+sudo podman pull docker.io/ceph/ceph:v14.2
+
+
+cat <<EOF | oc create -f -
```
Comment (BenHall) [inline environments/openshift-4-3/master/build/6_ocs-pull.sh:7]: This needs to happen when the scenario starts.

Comment (mulbc) [reply environments/openshift-4-3/master/build/6_ocs-pull.sh:11]: I feared so... ok I'll remove it then :/

**Comment by mulbc** [comment]
The production image has the local disks now and this can be merged

File: persistence-pathway.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Adding Persistence to OpenShift",
+  "icon": "fa-openshift",
+  "courses": [
+    {
+      "external_link": "https://learn.openshift.com/playgrounds/persistent-intro/",
```
Comment (jankleinert) [inline persistence-pathway.json:6]: This should be persistence, not playgrounds, right? In both instances @mulbc

Comment (mulbc) [reply persistence-pathway.json]: Fixed

**Pull request merged**